### ISSUE TITLE
Dash in model name

### ISSUE THREAD

**Issue by triccyx** [robotology-playground/sdf-modelica#6]
If dashes are present in the model name Modelica doesn't open the file.

Error:
<robot name="AB-01">

Correct:
<robot name="AB01">

Not a big problem just a reminder because Modelica errors are not very clear.

**Comment by traversaro** [comment]
Good point. We can develop a rule for escaping the identifiers that are accepted in XML attributes but rejected by Modelica.
